### PR TITLE
Reduce build time for cloudbuild

### PR DESCRIPTION
**Pull request by lzhecheng**
Cloudbuild used to serially build cloud-controller-manager and
cloud-node-manager images, which takes too much time. Now, with
variants.yaml, cloudbuild builds both images in parallel.

Signed-off-by: Zhecheng Li <zhechengli@microsoft.com>

#### What this PR does / why we need it:
Cloudbuild used to serially build cloud-controller-manager and
cloud-node-manager images, which takes too much time. Now, with
variants.yaml, cloudbuild builds both images in parallel.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by lzhecheng** [comment]
The doc about `variants.yaml` usage is here:
https://github.com/kubernetes/test-infra/blob/d22b2276527c3cbcdbf6bdb4234be2526ef10e4c/images/builder/README.md

File: variants.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+variants:
+  cloud-controller-manager-amd64:
+    CLOUD_BUILD_IMAGE: ccm-amd64
+  cloud-controller-manager-arm:
+    CLOUD_BUILD_IMAGE: ccm-arm
+  cloud-controller-manager-arm64:
+    CLOUD_BUILD_IMAGE: ccm-arm64
```
Comment (feiskyer) [inline variants.yaml:7]: would this build 3 ccm images or still merged into one multi-arch image?

Comment (lzhecheng) [reply variants.yaml:7]: Until now, ccm images are not merged into 1 multi-arch image and I didn't touch the script. Do you suggest doing so, like cnm?
https://console.cloud.google.com/gcr/images/k8s-staging-provider-azure/GLOBAL

Comment (lzhecheng) [reply variants.yaml]: Discussed offline. Let's make ccm images a multi-arch image.

**Review by feiskyer — COMMENTED** [review]
/lgtm

**Pull request merged**